media/js/firefox/personal.js

```diff
@@ -0,0 +1,28 @@
+$(function() {
+    'use strict';
+
+    var dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+    var $body = $('body');
+    var $flashTalkingPixel = $('<img />', {
+        width: '1',
+        height: '1',
+        src: 'https://servedby.flashtalking.com/spot/8/6247;40428;4669/?spotName=Mozilla_Download_Conversion',
+        class: 'flash-talking'
```

Comment:
Oh wow, that is interesting. To be honest, I do not even need it so, will just get rid of it.